src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoProcessorAttribute.cs

```diff
@@ -69,6 +71,56 @@ protected DittoProcessorAttribute()
         internal Type ContextType { get; set; }
 
         /// <summary>
+        /// Returns the MembershipHelper instance
+        /// </summary>
+        protected MembershipHelper Members
+        {
+            get { return new MembershipHelper(Umbraco); }
```

Comment:
I'm wondering if these should be stored on the processor context instead, and then these could maybe be convenience accessors?